server/server.js

```diff
@@ -40,6 +40,14 @@ const setupProxy = (fraPath, tilTarget, headers = undefined) =>
         },
     });
 
+const manifestMedEnv = () => {
```

Comment:
Vi kan evt senere ha en metode her som tar inn objekt, om det er flere filer som skal legges til

Reply:
Eller bare en stringarray for filer, og så kan key være lik siste segment i filstrengen.

Reply:
Eller at vi traverserer build mappen og legger til det som mangler, så slipper vi å spesifisere noe her :)